TestApexAssignment.cls

```diff
@@ -0,0 +1,49 @@
+@isTest(seeAllData=true)
```

Comment:
Instead of setting seeAllData=true you should create the test data for every test methods